proto/storage/report_configuration.proto

```diff
@@ -78,6 +78,17 @@ message NotifierConfiguration {
     oneof notifier_config {
         EmailNotifierConfiguration email_config = 1;
     }
+
+    // oneof is only used to work around the postgres store limitation. The oneof wrapper can be removed in future
+    // without any impact on the wiring.
+    //
+    // @mandar: "It appears that the DB upserts are encountering failures when handling unset string foreign keys.
+    // My educated assumption is that this might be attributed to the distinction between empty strings and null
+    // values. In Go, we do not differentiate whether an empty string signifies an expected empty value or an
+    // unset value, which is a requirement within SQL."
+    oneof ref {
+        string                         id           = 2 [(gogoproto.moretags) = 'sql:"fk(Notifier:id)"'];
```

Comment:
Is it guaranteed that we will only have a single notifier per report config? If we might have more than one, then the migration from a single fk to a 1:many relationship could be more complicated. So we might just want to start with a join table. Thoughts?

Reply:
Take a look at the `schema/report_configurations.go`

Reply:
Whoops for some reason I thought this was part of the ReportConfiguration proto and not its own.